Readme.md

```diff
@@ -9,9 +9,12 @@ Like Game Maker, ENIGMA comprises both an easy to use Drag & Drop system as well
 Although EDL adopts GML's very lax syntax in its scripting, the inputted code is actually parsed and translated into valid C++. This compilation improves size and performance of the language by incredible amounts, while almost magically maintaining its simplicity of use.
 
 ## Install
-In order to keep the ENIGMA repository small and fast binary files are excluded. To install you will need to obtain the binary files for your chosen platform. Please read the `building.md` file for more information.
+Please see http://enigma-dev.org/download.htm for installation instructions.
 
-### MACOSX
-After cloning the repository run the following command in terminal to install the main binary files
+## Documentation
+We have a community led Wiki project hosted on our site where users contribute function and other API documentation. All you will need to do is register for an account on our forums and you will be able to edit and make changes to the documentation pages.
```

Comment:
Please hyphenate community-led locally, so it'll submit with your next PR. Also, just say "all you need to do," as "will" seems to imply future work on our part.

Reply:
Completed in d2be0ac08ad1153bfc8daa687bc46aa991e8b268